graphics/grafx2/Portfile

```diff
@@ -19,18 +19,18 @@ long_description    GrafX2 is a bitmap paint program inspired by the Amiga progr
 homepage            http://grafx2.chez.com/
 distname            ${name}-v${version}
 master_sites        https://gitlab.com/GrafX2/grafX2/-/archive/v${version}/::gfx2 \
-                    sourceforge:project/recoil/recoil/4.2.0:recoil
+                    sourceforge:project/recoil/recoil/4.3.1:recoil
```

Comment:
Since the recoil version appears at least 4 times in the Portfile, I think a variable should be created to hold the `recoil_version` and then reference the variable. Put it just below the `version` line.

Reply:
you're right